WcaOnRails/app/views/registrations/edit_registrations.html.erb

```diff
@@ -42,7 +44,7 @@
 
         <tbody>
           <% registrations.each do |registration| %>
-            <tr id="registration-<%= registration.id %>" class="<%= registration.pending? ? "registration-pending" : "" %> <%= registration.accepted? ? "registration-accepted" : "" %>">
+            <tr id="registration-<%= registration.id %>" class="<%= registration.pending? ? "registration-pending" : "" %><%= registration.accepted? ? "registration-accepted" : "" %><%= registration.deleted? ? "registration-deleted" : "" %>">
```

Comment:
I'd put this in the Registration object. End result:

`registration-<%= registration.checked_status_name %>`